### ISSUE TITLE
Safety Info - Marine

### ISSUE THREAD

**Issue by kmranjo** [weather-gov/weather.gov#1083]
Generated from #777

To include all marine hazard products

**Comment by coreypieper** [comment]
@kmranjo, have you already made content for this? I'm in no hurry, just working through all of the statues for safety today.

**Comment by kmranjo** [comment]
Nope, haven't picked up Marine products yet, I think we are tackling them in a later stage

**Comment by coreypieper** [comment]
Sounds good!
